### PR TITLE
Trivial fixups

### PR DESCRIPTION
**Pull request by levitte**

### CODE REVIEW

File: ssl/statem/extensions.c [hunk 1]
```diff
@@ -1533,7 +1533,7 @@ int tls_psk_do_binder(SSL *s, const EVP_MD *md, const unsigned char *msgstart,
         void *hdata;
 
         hdatalen = BIO_get_mem_data(s->s3->handshake_buffer, &hdata);
-        if (hdatalen <= 0) {
+        if (hdatalen == 0) {
```
Comment (t-j-h) [inline ssl/statem/extensions.c]: Why this change? BIO_ctrl can return negative values ...

Comment (levitte) [reply ssl/statem/extensions.c]: On line 1532, `hdatalen` is defined as `size_t`...  I'll think of a different solution.

File: ssl/statem/statem_lib.c [hunk 2]
```diff
@@ -203,9 +203,10 @@ static int get_cert_verify_tbs_data(SSL *s, unsigned char *tls13tbs,
         *hdatalen = TLS13_TBS_PREAMBLE_SIZE + hashlen;
     } else {
         size_t retlen;
+        long retlen_l;
 
-        retlen = BIO_get_mem_data(s->s3->handshake_buffer, hdata);
-        if (retlen == 0) {
+        retlen = retlen_l = BIO_get_mem_data(s->s3->handshake_buffer, hdata);
+        if (retlen <= 0) {
```
Comment (t-j-h) [inline ssl/statem/statem_lib.c]: retlen_l comparison here ...

Comment (levitte) [reply ssl/statem/statem_lib.c]: Urgh...  starting to be way too late at night...

File: ssl/statem/extensions.c [hunk 3]
```diff
@@ -1530,10 +1530,12 @@ int tls_psk_do_binder(SSL *s, const EVP_MD *md, const unsigned char *msgstart,
      */
     if (s->hello_retry_request == SSL_HRR_PENDING) {
         size_t hdatalen;
+        long hdatalen_l;
         void *hdata;
 
-        hdatalen = BIO_get_mem_data(s->s3->handshake_buffer, &hdata);
-        if (hdatalen <= 0) {
+        hdatalen = hdatalen_l =
```
[the remark this answers is not in the document]
Comment (levitte) [reply ssl/statem/extensions.c]: Not in my opinion

**Comment by levitte** [comment]
Ping

**Review by paulidale — APPROVED** [review]
